misk/src/test/kotlin/misk/web/interceptors/RequestLoggingInterceptorTest.kt

```diff
@@ -61,25 +73,20 @@ internal class RequestBodyLoggingInterceptorTest {
       "ExcludeBodyRequestLoggingAction principal=caller",
       "ExcludeBodyRequestLoggingAction principal=caller time=100.0 ms code=200"
     )
-    assertThat(logCollector.takeMessages(RequestBodyLoggingInterceptor::class)).isEmpty()
+    assertThat(logCollector.takeMessages(RequestLoggingInterceptor::class)).isEmpty()
   }
 
   @Test
-  fun exceptionThrown() {
```

Comment:
why did you shift the order of these tests?

Reply:
Thought the ordering made more sense, I can undo